File: arrays/median_of_two_sorted_arrays.py
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        combined = sorted(nums1 + nums2)
        length = len(combined)
        middle = length // 2

        if length % 2 == 0:
            return (combined[middle] + combined[middle - 1]) / 2.0
        else:
            return combined[middle]

File: arrays/median_of_two_sorted_arrays.rb
# @param {Integer[]} nums1
# @param {Integer[]} nums2
# @return {Float}
def find_median_sorted_arrays(nums1, nums2)
    combined = (nums1 + nums2).sort
    len = combined.size
    middle_ind = len / 2

    if len.even?
        return ((combined[middle_ind]) + (combined[(middle_ind - 1)])) / 2.00000
    else
        return combined[middle_ind]
    end
end